### ISSUE TITLE
Add -l -v to list tracepoint arguments

### ISSUE THREAD

**Issue by brendangregg** [iovisor/bpftrace#323]
If I add a -v to -l, it should print arguments. Eg:

```
# bpftrace -lv 'tracepoint:raw_syscalls:sys_enter'
tracepoint:raw_syscalls:sys_enter
```

That should print the arg members that are accessible (with args->) from:

```
cat /sys/kernel/debug/tracing/events/raw_syscalls/sys_enter/format 
name: sys_enter
ID: 19
format:
	field:unsigned short common_type;	offset:0;	size:2;	signed:0;
	field:unsigned char common_flags;	offset:2;	size:1;	signed:0;
	field:unsigned char common_preempt_count;	offset:3;	size:1;	signed:0;
	field:int common_pid;	offset:4;	size:4;	signed:1;

	field:long id;	offset:8;	size:8;	signed:1;
	field:unsigned long args[6];	offset:16;	size:48;	signed:0;

print fmt: "NR %ld (%lx, %lx, %lx, %lx, %lx, %lx)", REC->id, REC->args[0], REC->args[1], REC->args[2], REC->args[3], REC->args[4], REC->args[5]
```

So I can stop cat-ing /sys. :)

Oh, Sasha already did this for bcc:

```
# /usr/share/bcc/tools/tplist -v raw_syscalls:sys_enter
raw_syscalls:sys_enter
    long id;
    unsigned long args[6];
```

That output is perfect (just missing the "tracepoint:" probe type prefix).

**Comment by danobi** [comment]
+1, was just about to file an issue for this.

**Comment by brendangregg** [comment]
fixed by #375
